Library/src/Systems/Maintenance/MaintenanceSystem.cpp

```diff
@@ -31,23 +31,23 @@ namespace csp::systems
 
 MaintenanceSystem::MaintenanceSystem() : SystemBase(), MaintenanceAPI(nullptr)
 {
-	Running = true;
+	AllowMaintenanceInfoRequests = true;
 }
 
 MaintenanceSystem::MaintenanceSystem(csp::web::WebClient* InWebClient) : SystemBase(InWebClient)
 {
-	MaintenanceAPI = CSP_NEW chs::MaintenanceApi(InWebClient);
-	Running		   = true;
+	MaintenanceAPI				 = CSP_NEW chs::MaintenanceApi(InWebClient);
+	AllowMaintenanceInfoRequests = true;
 }
 
 MaintenanceSystem::~MaintenanceSystem()
 {
 	CSP_DELETE(MaintenanceAPI);
 }
 
-void MaintenanceSystem::GetMaintenanceInfo(MaintenanceInfoCallback Callback)
+void MaintenanceSystem::GetMaintenanceInfo(const csp::common::String& MaintainanceURL, MaintenanceInfoCallback Callback)
 {
-	if (Running == true)
+	if (AllowMaintenanceInfoRequests == true)
```

Comment:
Could we emit a message to the log if `AllowMaintenanceInfoRequests` is false in this function? I'm imagining a client application developer calling this function, getting it wrong the first time and then wondering why they don't get callbacks on subsequent invocations. In that instance, giving them feedback via the log feels right?

Reply:
Done

Reply:
Thanks!